Doc/whatsnew/2.3.rst

```diff
@@ -1391,8 +1391,8 @@ complete list of changes, or look through the CVS logs for all the details.
   automatically generate a usage message.  See the following section for  more
   details.
 
-* The old and never-documented :mod:`linuxaudiodev` module has been deprecated,
-  and a new version named :mod:`ossaudiodev` has been added.  The module was
+* The old and never-documented :mod:`!linuxaudiodev` module has been deprecated,
+  and a new version named :mod:`!ossaudiodev` has been added.  The module was
```

Comment:
We can keep this ref in 3.12, it was removed in 3.13: https://peps.python.org/pep-0594/#ossaudiodev


```suggestion
  and a new version named :mod:`ossaudiodev` has been added.  The module was
```